### ISSUE TITLE
Contact Tracing article - 13.4 background Bluetooth services info source?

### ISSUE THREAD

**Issue by KaneCheshire** [NSHipster/articles#721]
Hey,

Just looking through your post, it mentions: 

>As of iOS 13.4, 3rd-party apps lack the ability to advertise Bluetooth services in the background.

Where has that information come from? There's no link to any docs and I've had a good Google this morning and couldn't find anything.

Also, sort of related, I found some new background behaviour with force-quitting the app in iOS 13.4 which I can't find referenced anywhere so not sure if it's a bug or expected behaviour.

Thanks!

**Comment by mattt** [comment]
@KaneCheshire Thank you for raising this issue. These limitations were claimed by one of the projects I found in my research, but going over my notes, I can't find the original source. I believe it was https://github.com/DP-3T/dp3t-sdk-ios, but I can't find any record of that there (it looks like they've force-pushed a hard reset to the project since the Apple / Google announcement, so I can't look back through the history to check).

After trying to fact-check this now, I wasn't able to substantiate this claim, so I removed the statement with 988c1fc9f430fbb8d8f8114ebd26fa065d367d4b.

For the record: 

According to the [Core Bluetooth Programming Guide](https://developer.apple.com/library/archive/documentation/NetworkingInternetWeb/Conceptual/CoreBluetooth_concepts/CoreBluetoothBackgroundProcessingForIOSApps/PerformingTasksWhileYourAppIsInTheBackground.html), you **can** advertise and discover Bluetooth services in the background. However, there are some unspecified restrictions on the limitations imposed by iOS.

My best guess is that the original statement about these limitations had to do with either the background transmission being too infrequent to be effective or the inability for the app to regenerate the identifiers in the background. Either way, my interpretation of that as a fundamental prohibition of background processing was in error.
